addons/packages/kapp-controller/0.30.0/bundle/config/overlays/update-deployment.yaml

```diff
@@ -29,9 +29,12 @@ spec:
           #@overlay/match by="name"
           - name: KAPPCTRL_API_PORT
             value: #@ str(values.kappController.deployment.apiPort)
-      #@ if/end values.kappController.deployment.hostNetwork:
+      #@ if values.kappController.deployment.hostNetwork:
       hostNetwork: #@ values.kappController.deployment.hostNetwork
+      dnsPolicy: ClusterFirstWithHostNet
+      #@ end
       #@ if/end values.kappController.deployment.dnsPolicy:
+      #@yaml/map-key-override
```

Comment:
Does this allow override if `values.kappController.deployment.dnsPolicy` is not null or empty?

Reply:
Yes, if dnsPolicy is set to another value, it will override.